test/util/utils.js

```diff
@@ -109,7 +109,16 @@ exports.getFilesChangedInPR = function getFilesChangedInPR() {
       console.log('>>>>> Files changed in this PR are as follows:')
       console.log(filesChanged);
       swaggerFilesInPR = filesChanged.split('\n').filter(function (item) {
-        return (item.match(/.*\/swagger\/*/ig) !== null);
+        if (item.match(/.*json$/ig) == null) {
```

Comment:
Just to understand, we're going to ignore all json files here and work with .md files only?